frontend/packages/dev-console/src/components/pipelines/pipelinesResources/PipelinesResourceForm.tsx

```diff
@@ -46,15 +46,19 @@ const PipelineResourceForm: React.FC<PipelineResourceFormProps> = ({
       type: 'cluster',
       param: {
         url: '',
+        username: '',
         password: '',
         insecure: false,
       },
+      secret: {
```

Comment:
let us follow the same convention that is provided in the docs
```suggestion
      secrets: {
```